reference/opt/shaders-msl/vulkan/vert/multiview.multiview.nocompat.vk.vert

```diff
@@ -0,0 +1,32 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct MVPs
+{
+    float4x4 MVP[2];
+};
+
+struct main0_out
+{
+    float4 gl_Position [[position]];
+    uint gl_Layer [[render_target_array_index]];
+};
+
+struct main0_in
+{
+    float4 Position [[attribute(0)]];
+};
+
+vertex main0_out main0(main0_in in [[stage_in]], constant uint* spvViewMask [[buffer(24)]], constant MVPs& _19 [[buffer(0)]], uint gl_InstanceIndex [[instance_id]])
+{
+    main0_out out = {};
+    uint spvMaxViewIndex = 31 - clz(*spvViewMask);
+    uint gl_ViewIndex = gl_InstanceIndex % spvMaxViewIndex;
```

Comment:
There is an alternative here to render all instances for a layer, and this style renders all views in one go. Not sure which style is best from a caching POV, but optimizing for attribute read caching which this code does might be best here.